src/sentry/demo/data_population.py

```diff
@@ -1043,6 +1043,47 @@ def populate_generic_error(self, project: Project, file_path, dist_number, start
             self.safe_send_event(local_event)
         self.log_info("populate_generic_error.finished")
 
+    def populate_generic_transaction(
+        self, project: Project, file_path, dist_number, starting_release=0
+    ):
+        """
+        This function populates a single transaction
+        Occurrance times and durations are randomized
+        """
+        transaction = get_event_from_file(file_path)
+
+        self.log_info("populate_generic_transaction.start")
+
+        for (timestamp, day) in self.iter_timestamps(dist_number, starting_release):
+            transaction_user = self.generate_user()
+            trace_id = uuid4().hex
+            release = get_release_from_time(project.organization_id, timestamp)
+            release_sha = release.version
+
+            old_span_id = transaction["contexts"]["trace"]["span_id"]
+            ios_span_id = uuid4().hex[:16]
+            duration = self.gen_frontend_duration(day)
+
+            trace = {"trace_id": trace_id, "span_id": ios_span_id}
```

Comment:
@roggenkemper do we actually need this `trace` dict? I believe `update_context` doesn't require `trace` and will automatically create a trace and span for it.

Reply:
I think you're right.